### PR TITLE
Let IRC bot know of publishes

### PR DESCRIPTION
**Pull request by errietta**
Talks to IRC bot - requires https://github.com/mozilla/webmaker-app-bot/pull/1

### CODE REVIEW

**Comment by errietta** [comment]
R? @k88hudson

File: server/routes/webmaker-app.js [hunk 1]
```diff
@@ -16,11 +17,27 @@ var webmakerVersion = require('../../package.json').dependencies.webmaker;
 if (!habitat.get('FIREBASE_ENDPOINT')) throw new Error('You must configure FIREBASE_ENDPOINT in .env');
 
 var firebase = new Firebase(habitat.get('FIREBASE_ENDPOINT') + '/apps');
+var ircBot = habitat.get('BOT_ENDPOINT');
 
 module.exports = function (req, res, next) {
     var user = req.session && req.session.user;
     var baseDir = 'p';
 
+    function tellIRC (evt, data) {
```
Comment (k88hudson) [inline server/routes/webmaker-app.js]: You need to do a check somewhere if `ircBot` is truthy; you can either early return in this function, or do a conditional check where you run `tellIRC`

File: server/routes/webmaker-app.js [hunk 2]
```diff
@@ -16,11 +17,27 @@ var webmakerVersion = require('../../package.json').dependencies.webmaker;
 if (!habitat.get('FIREBASE_ENDPOINT')) throw new Error('You must configure FIREBASE_ENDPOINT in .env');
 
 var firebase = new Firebase(habitat.get('FIREBASE_ENDPOINT') + '/apps');
+var ircBot = habitat.get('BOT_ENDPOINT');
```
Comment (k88hudson) [inline server/routes/webmaker-app.js]: if you need to have this in a variable, i would go with constant-style `var BOT_ENDPOINT = habitat.get('BOT_ENDPOINT')`

File: server/routes/webmaker-app.js [hunk 3]
```diff
@@ -69,6 +86,9 @@ module.exports = function (req, res, next) {
             res.send({
                 url: habitat.get('PUBLISH_URL') + '/' + dir
             });
+
+            tellIRC('publish', json);
```
Comment (k88hudson) [inline server/routes/webmaker-app.js]: right now this only runs in the fake auth case, you will also want to run this in the real case when the s3 tasks have completed

**Comment by k88hudson** [comment]
A few comments, this is sweet! So excited :smile: 

Also, in terms of configuration and deployment:
* a are we deploying both a staging/mofodev instance? @thisandagain is this already done/does this need to be done (we can ping jp to do this if needed). We should add those endpoints to the `config/` folder
* I'm thinking we might want to just set the default value to `false` or the mofodev endpoint, as long as we're OK with people hitting it all the time with fake routes

Also,@errietta can you file a ticket to add authentication at some point?

**Comment by thisandagain** [comment]
I think all we need is a staging instance, but this still needs to be done with JP. If you could help get the instance setup I'd appreciate it.

**Comment by errietta** [comment]
@k88hudson auth issue https://github.com/mozilla/webmaker-app/issues/813

**Comment by k88hudson** [comment]
Awesome :dancer:

**Pull request merged**